script.js

```diff
@@ -122,8 +123,10 @@ const Weathertron = {
 		}
 
 		// Render the message
-		let classes = 'weathertron-forecast-empty is-' + messageType;
-		let messageEl = '<div class="' + classes + '">' + message + '</div>';
+		// let classes = 'weathertron-forecast-empty is-' + messageType;
+		let classes = `weathertron-forecast-empty is- ${messageType}`;
```

Comment:
You can't have a space between `is-` and `${messageType}` or else you end up with `is- error` which is two different classes, `.is-` and `.error`